### PR TITLE
Elm-UI Hook for beginners

### PR DESCRIPTION
**Pull request by 13r0ck**
This would have helped me understand the Elm-UI hook. Totally open to changes though!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,20 @@ g =
...
+The hook is just a wrapper arround Elm-UI `Element's. A new wrapper
+must be created for each element that you would like to animate.
+Though because Elm-UI `Element`s can all be wrapped in either `el`,
+`row` or `column` that is usually all we need.
+
+To do that first we need to add a helper function so that wrappers can
+can be created more easily. Just import `elm-simple-animation` with
+```
+import Simple.Animation as Animation exposing (Animation)
+import Simple.Animation.Animated as Animated
```
Comment (andrewMacmurray) [inline README.md]: Maybe take the `as Animated` bit off so the code below will work

File: README.md [hunk 2]
```diff
@@ -128,18 +142,23 @@ animatedUi =
...
+Finally we need to wrap the Elm-UI `Element`s!
+Because you will likely still need to create non-animated
+`Element`s, lets give these a similar name `ael` and `acolumn`
+for "animated element" and "animated column". Just paste these into your file
 
 ```elm
-el : Animation -> List (Element.Attribute msg) -> Element msg -> Element msg
-el =
+ael : Animation -> List (Element.Attribute msg) -> Element msg -> Element msg
+ael =
```
Comment (andrewMacmurray) [inline README.md:152]: would you mind calling these `animatedEl` and `animatedColumn`? I feel it might be clearer for newcomers what they are

**Comment by andrewMacmurray** [comment]
Thanks for this @13r0ck, just left a few comments. Would you mind taking a look and then happy to merge in 👍

File: README.md (hunk 1 above)
Comment (13r0ck) [reply README.md]: Oops! Thanks!

File: README.md (hunk 2 above)
Comment (13r0ck) [reply README.md:152]: Makes sense! Just made the changes!

**Comment by 13r0ck** [comment]
Made the changes! Thanks for the review! :smile:

**Comment by andrewMacmurray** [comment]
Great, thanks for this 👍

**Comment by andrewMacmurray** [comment]
closes #12

**Pull request merged**